### PR TITLE
[MODULAR] Fighterslam - Captain Jacket

### PR DESCRIPTION
**Pull request by fighterslam**
## About The Pull Request
Ports Beestation's Captain Jacket from https://github.com/BeeStation/BeeStation-Hornet/pull/2621 and adds it to the Command Vendor. 
I also cleaned up the description a bit to be more descriptive and have proper grammar/spelling. 

## How This Contributes To The Skyrat Roleplay Experience
Adds a new jacket for the Captain to use. More clothing variety! Fairly self explanatory.

## Changelog

:cl:
add: Ports the Captain's Jacket over from Beestation, now found in the Command Vendor!
/:cl:

### CODE REVIEW

**Comment by GoldenAlpharex** [comment]
This needs in-game pictures of the sprite.

**Comment by fighterslam** [comment]
The original PR linked has in-game pictures. Here's the link they used to reference the Captain's Jacket: https://gyazo.com/12418626e3dae6eaeec887c0612455d8

**Comment by GoldenAlpharex** [comment]
> The original PR linked has in-game pictures. Here's the link they used to reference the Captain's Jacket: https://gyazo.com/12418626e3dae6eaeec887c0612455d8

Yeah, but the reason why I ask those is to ensure that you tested them, and to know how they look in our codebase.

**Comment by fighterslam** [comment]
Here's the in-game images on my branch.  Anything else?

image
image
image
image

**Comment by GoldenAlpharex** [comment]
> Here's the in-game images on my branch. Anything else?
> 
> image image image image

It looks nice, however, the gold color does not match the current gold color of the Captain's attire. Once you fix that, though, I'd be down for it.

**Comment by fighterslam** [comment]
image
image
image
image
image
 New sprites.

File: code/modules/clothing/suits/armor.dm [hunk 1]
```diff
@@ -259,6 +259,12 @@
 	inhand_icon_state = "capspacesuit"
 	body_parts_covered = CHEST|GROIN|ARMS
 
+/obj/item/clothing/suit/armor/vest/capcarapace/jacket
```
Comment (tf-4) [inline code/modules/clothing/suits/armor.dm]: Modularise

Comment (ghost) [reply code/modules/clothing/suits/armor.dm]: helped walk them thru it, should be modular now 👍

**Review by GoldenAlpharex — APPROVED** [review]
Yeah I can get behind that.

**Pull request merged**